charged-ieee/lib.typ

```diff
@@ -173,12 +173,14 @@
     scope: "parent",
     clearance: 30pt,
     {
-      {
+      show std.title: t => {
         set align(center)
         set par(leading: 0.5em)
-        set text(size: 24pt)
-        block(below: 8.35mm, title)
+        set text(size: 24pt, weight: "regular")
+        set block(below: 8.35mm)
+        t
```

Comment:
it would be be better to split this into four show-set rules, as they compose better. here it does not make a big difference because the one and only title follows immediately, but it's still good practice.